rllib/algorithms/algorithm.py

```diff
@@ -846,23 +859,31 @@ def step(self) -> ResultDict:
             ), "Algorithm.evaluate() needs to return a dict."
             results.update(self.evaluation_metrics)
 
-        if hasattr(self, "workers") and isinstance(self.workers, WorkerSet):
-            # Sync filters on workers.
+        # Sync filters on workers.
```

Comment:
All algos now have a worker set attribute (the ones that didn't were moved to rllib_contrib).